packages/nodes-base/.eslintrc.js

```diff
@@ -55,7 +55,8 @@ module.exports = {
 		'@typescript-eslint/restrict-template-expressions': 'off',
 		'@typescript-eslint/return-await': 'off',
 		'@typescript-eslint/unbound-method': 'off',
-		'@typescript-eslint/ban-ts-comment': ['warn', { 'ts-ignore': true }],
+		// '@typescript-eslint/ban-ts-comment': ['warn', { 'ts-ignore': true }],
+		'@typescript-eslint/ban-ts-comment': 'off',
```

Comment:
we shouldn't be turning this rule off, as that will only encourage the use of `@ts-ignore`, which defeats the purpose of using a typed language.

Reply:
it is temporary, I will re enable it after completion

Reply:
ah okay. in that case please mark the PR as Draft. 👍🏽